week03_model_free/README.md

```diff
@@ -18,37 +19,8 @@
 ### Assignments
 
 Just as usual, start with 
-- `seminar_qlearning.ipynb` 
-[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/yandexdataschool/Practical_RL/blob/master/week03_model_free/seminar_qlearning.ipynb)
+- [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/yandexdataschool/Practical_RL/blob/master/week03_model_free/seminar_qlearning.ipynb)
```

Comment:
I guess that the link should be to a specific commit hash or the `spring20` branch, not `master`.